lib/PuppeteerSharp/CDPSession.cs

```diff
@@ -101,27 +103,19 @@ internal CDPSession(IConnection connection, TargetType targetType, string sessio
         /// <returns>The task.</returns>
         public async Task<T> SendAsync<T>(string method, dynamic args = null)
         {
-            var content = await SendAsync(method, true, args).ConfigureAwait(false);
-            return JsonConvert.DeserializeObject<T>(content);
-        }
+            JObject content = await SendAsync(method, args);
```

Comment:
We can turn this into an arrow function.
`=> (await SendAsync(method, args).ConfigureAwait(false)).ToObject<T>();`

Remember to always use `ConfigureAwait(false)`

Reply:
When you try it this way, the ambiguity of the call means that it call's itself and causes a stack overflow. Unless you know of a way to force the compiler to use the 'JObject' version of the method ?

Reply:
Nevermind, I think at runtime this resolves correctly

Reply:
Having said it's resolved I get partial unit test failure caused by unwanted recursion, I'll leave as is, maybe you know of a way to resolve it.

Reply:
I'm good with this